apps/teamsjs-cc-app/tabs/src/components/capabilities/pages/ReturnFocusToSearchBar.tsx

```diff
@@ -0,0 +1,38 @@
+import * as Fluent from "@fluentui/react-northstar";
+
+import { app, pages } from "@microsoft/teams-js";
+
+/**
+ * This component returns a button which returns focus to search bar.
+ */
+export const ReturnFocusToSearchBar = () => {
+  // cCheck if app is initialized;
+  if (app.isInitialized()) {
+    return (
+      <>
+        {pages.isSupported() && (
+          <Fluent.Segment className="ui-pagessegment">
+            <Fluent.Header content="Return Focus To Search Box" as="h2" />
+            <Fluent.Flex gap="gap.small" vAlign="center">
+              <Fluent.Text
+                className="ui-pagestext"
+                content="Returns focus to the host's Search box. (Curently works only in teams)"
```

Comment:
Same question about following up with non-teams hosts

Reply:
same as the previous response